docs/getting-started/generating-deployment-profiles.md

```diff
@@ -0,0 +1,222 @@
+It is possible to manually create these files, but a section below explains how to do this
+more easily for organizations that don't have deep plist file-format expertise.
+
+### Windows
+
+The Windows deployments are stored in the registry. User deployments are stored at:
+```
+HKEY_CURRENT_USER\SOFTWARE\Policies\Rancher Desktop\Defaults
+HKEY_CURRENT_USER\SOFTWARE\Policies\Rancher Desktop\Locked
+```
+
+And the system deployments are stored at:
+```
+HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Rancher Desktop\Defaults
+HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Rancher Desktop\Locked
+```
+
+The structure of the registry instances follows the structure of the JSON files for
+the other platforms. The name of a non-leaf node in the JSON structure (a name to the
+left of a ":") will appear as a registry key. Leaf nodes appear as key-value pairs
+in the right-hand pane of a key's view, where the key is the field's name.
+And hash-like objects (like `WSL.integrations`) also appear as key-value pairs,
+in this case for the key `...\WSL\integrations`.
+
+And if you're coding against the registry, names are always case-insensitive. Feel
+free to use all lower-case, UPPER-CASE, or spOnGeBoBCasE.
+
+## Generating deployments
+
+The easiest way to do this is on a system already running Rancher Desktop, mainly for its
+`rdctl` command-line tool, and the default list of settings.
+
+First, you're going to want to verify that the deployments you create make sense on a
+working Rancher Desktop, so the rest of this document assumes you'll have access to it.
+Rancher Desktop runs on air-gapped systems, so if you're creating deployments for
+security-related reasons, security shouldn't be a concern while creating them.
+
+We also assume you've installed the `jq` json tool. If you have a preferred variant, simply
+substitute it for `jq` in the following text.
+
+### Crafting JSON deployment files
+
+Let's assume you want to create both default-value and locked-value deployment profiles.
+
+If you want to start with Rancher Desktop's default values, and don't have your own
+workloads you need to preserve, you can run `rdctl factory-reset` first.
+
+Start up Rancher Desktop, and run the following commands in your preferred shell:
+
+```
+rdctl list-settings | jq . > working-defaults.txt
+rdctl list-settings | jq . > working-locked.txt
+```
+
+If you're only making one type of profile, you'll only need one of the above lines, of course.
+It also doesn't make sense to create both a user and system profile for the same type of
+deployment. If Rancher Desktop sees a system-profile, it won't load the user-profile of the same kind.
+However, if Rancher Desktop finds, for example, a user-defaults and a system-locked profile, but no
+system-defaults profile, it will load the user-defaults and system-locked profiles.
+
+Next, you can edit each of the JSON files to contain only the values you care about. You can
+use the `jq` command to validate the final outputs:
```

Comment:
```suggestion
If you're only making one type of profile, you'll only need one of the above lines, of course.
It also is not necessary to create both a user and system profile for the same type of deployment. If Rancher Desktop sees a system-profile, it won't load the user-profile of the same kind. However, if Rancher Desktop finds, for example, a user-defaults and a system-locked profile, but no system-defaults profile, it will load the user-defaults and system-locked profiles.

Next, you can edit each of the JSON files to contain only the values you care about. You can use the `jq` command to validate the final outputs:
```